app/preprints/new-version/route.ts

```diff
@@ -0,0 +1,78 @@
+import Store from '@ember-data/store';
+import Route from '@ember/routing/route';
+import RouterService from '@ember/routing/router-service';
+import { inject as service } from '@ember/service';
+import Toast from 'ember-toastr/services/toast';
+import Intl from 'ember-intl/services/intl';
+
+import Theme from 'ember-osf-web/services/theme';
+import MetaTags, { HeadTagDef } from 'ember-osf-web/services/meta-tags';
+import PreprintModel from 'ember-osf-web/models/preprint';
+
+export default class PreprintNewVersionRoute extends Route {
+    @service store!: Store;
+    @service theme!: Theme;
+    @service router!: RouterService;
+    @service metaTags!: MetaTags;
+    @service toast!: Toast;
+    @service intl!: Intl;
```

Comment:
If we decided we don't need the stuff in the comment, you probably don't need this either.